docs/root/intro/version_history.rst

```diff
@@ -48,6 +48,8 @@ Version history
 * upstream: added configuration option to the subset load balancer to take locality weights into account when
   selecting a host from a subset.
 * access log: added RESPONSE_DURATION and RESPONSE_TX_DURATION.
+* cluster: added :ref:`option <envoy_api_field_Cluster.CommonLbConfig.update_merge_window>` to merge health check/weight/metadata
+  updates within the given duration.
```

Comment:
nit: reflow to 100 cols. Please also alpha sort the subsystem (can you also fix the others that crept in out of order)